source/dofs/dof_renumbering.cc

```diff
@@ -1197,9 +1197,10 @@ namespace DoFRenumbering
     template <int dim, class CellIteratorType>
     types::global_dof_index
     compute_hierarchical_recursive (const types::global_dof_index         next_free_dof_index,
-                                    std::vector<types::global_dof_index> &new_indices,
+                                    const types::global_dof_index         my_starting_index,
                                     const CellIteratorType               &cell,
-                                    const IndexSet                       &locally_owned_dof_indices)
+                                    const IndexSet                       &locally_owned_dof_indices,
+                                    std::vector<types::global_dof_index> &new_indices)
     {
       types::global_dof_index current_next_free_dof_index = next_free_dof_index;
```

Comment:
The argument ``next_free_dof_index`` is now an offset and not a dof index. Can you rename the argument accordingly?